macOS.md

```diff
@@ -1,21 +1,74 @@
 ---
 layout: home
-title: Implementation guide (iOS)
+title: Implementation guide (macOS/iOS)
 nav_order: 6
 nav_titles: true
 titles_max_depth: 2
 ---
 
 This guide is for **application developers** working to add native MPTCP support
-when running on iOS.
+when running on macOS/iOS. Be careful that some code presented on this page is
+only supported on iOS.
 
-## MPTCP connection
+{: .warning }
+Note however that only client side MPTCP seems supported for the time being 
+on macOS/iOS.
+
+## MPTCP with the Network framework
```

Comment:
I think it would be better to move this section after `MPTCP with URLSession (iOS only)`. My understanding is that `URLSession` is more popular because most of the communications are done on top of HTTP(s). Here, it is a lower level, only for specific needs.

Reply:
Yes, the `Network` framework seems targeted for developers who want to have a finer grained approach on the networking part. Most of the people creating an application will surely use the `URLSession` API (which is built atop `Network`), or will create their own library atop `Network` I imagine, but it will be less common as you say.

Reply:
Good, then better to switch the order as you recently did!